docs/Actions/inject-data.md

```diff
@@ -0,0 +1,273 @@
+---
+id: inject-data
+title: Inject Data
+sidebar_label: Inject Data
+---
+
+> Audience: [`Business Users`](/docs/audience#business-users)<br/>
+> Skill Prerequisites: `None`
+
+An action that lets you add data as tokens to the current execution context. Inject data can be used to load data that other actions other actions down the stack need.
+
+## `Typical Use Cases`
+
+- Generate a new token
+- Concatenate 2 tokens
+- Overwrite a token
+
+## `Don't use it to`
+
+- Execute a Variables token - use [Apply Tokens](/docs/Actions/apply-tokens.md) or [Execute Tokens](/docs/Actions/execute-tokens.md) actions instead
+- Apply tokens inside a text from different sources - use [Apply Tokens](/docs/Actions/apply-tokens.md) or [Execute Tokens](/docs/Actions/execute-tokens.md) actions instead
+- Use a token into another action list, only use it for the current one - store the data in a [hidden field](/docs/Actions/hidden-field.md), cookies or server session instead
```

Comment:
Not sure I understand. The way I read is is "Don't use it to use tokens into another action list". What is an action list? What other action lists might be there? What does action lists have to do with hidden fields?

Reply:
I redrafted it and changed "action list" with "execution context". Hopefully it's more clear.